tests/components/generic/test_config_flow.py

```diff
@@ -136,10 +140,11 @@ async def test_form_only_svg_whitespace(hass, fakeimgbytes_svg, user_flow):
     respx.get("http://127.0.0.1/testurl/1").respond(stream=fakeimgbytes_wspace_svg)
     data = TESTDATA.copy()
     data.pop(CONF_STREAM_SOURCE)
-    result2 = await hass.config_entries.flow.async_configure(
-        user_flow["flow_id"],
-        data,
-    )
+    with patch("homeassistant.components.generic.async_setup_entry", return_value=True):
+        result2 = await hass.config_entries.flow.async_configure(
+            user_flow["flow_id"],
+            data,
+        )
     assert result2["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
```

Comment:
We should always block till done after the last step before creating the entry inside the patch context manager, to make sure that all tasks are done.

Reply:
Done 6a0829b